modules/console/pages/config/security/authentication.adoc

```diff
@@ -1,61 +1,367 @@
+    E & H --> I[Store JWT as secure cookie]
+    I --> J[User makes API call]
+
+    %% API selection
+    J --> K{Which API?}
+    K -- "Kafka API" --> L[Check Kafka impersonation setting]
+    K -- "Schema registry API" --> M[Check schema registry impersonation setting]
+    K -- "Admin API" --> N[Check admin API impersonation setting]
+
+    %% Kafka API branch
+    subgraph KafkaFlow [Kafka API flow]
+      L --> O{User impersonation enabled?}
+      O -- Yes --> P["Forward JWT credentials\n(SASL-OAUTHBEARER or SASL-SCRAM)"]
+      O -- No --> Q[Use static credentials\n+ enforce console RBAC]
+    end
+
+    %% Schema registry branch
+    subgraph SchemaRegistryFlow [Schema registry API flow]
+      M --> R{User impersonation enabled?}
+      R -- Yes --> S[Forward JWT credentials\nheader-based authentication]
+      R -- No --> T[Use static credentials\n+ enforce console RBAC]
+    end
+
+    %% Admin API branch
+    subgraph AdminAPIFlow [Admin API flow]
+      N --> U{User impersonation enabled?}
+      U -- Yes --> V[Forward JWT credentials\nheader-based authentication]
+      U -- No --> W[Use static credentials\n+ enforce console RBAC]
+    end
+....
+
+=== OIDC authentication
+
+When using OIDC, the user is redirected to an external identity provider, such as Google, for authentication. Upon successful login, the provider issues an ID token which is then captured and encapsulated within a JWT. This token is used in the following ways:
+
+* **Kafka authentication:**
+  The ID token is mapped to SASL-OAUTHBEARER, allowing Kafka to authenticate requests under the user's identity.
+
+* **HTTP API authentication:**
+  The ID token is used in header-based authentication (`Authorization: Bearer <id-token>`) in the Schema Registry and Admin API.
 
-* xref:./azure-ad.adoc[Microsoft Entra ID]
-* xref:./google.adoc[Google]
-* xref:./github.adoc[GitHub]
-* xref:./keycloak.adoc[Keycloak]
-* xref:./okta.adoc[Okta]
-* xref:./generic-oidc.adoc[Generic OIDC]
+=== Basic authentication
 
-You can use one or more login providers at the same time. To enable SSO authentication,
-you must create an OAuth application for your organization first. Refer to the respective
-documentation page for guidance to set up your desired identity provider in Redpanda Console.
-Afterwards, you can configure your identity provider in Redpanda Console by providing the
-clientId and clientSecret in the configuration block for your provider. The configuration
-to add Google login support looks like this:
+For basic authentication, the user submits a username and password directly to Redpanda Console. These credentials are validated and encapsulated within a JWT, which is then used as follows:
+
+* **Kafka authentication:**
+  The provided credentials are mapped to SASL-SCRAM, ensuring that Kafka operations execute with the correct user identity.
+
+* **HTTP API authentication:**
+  The JWT supports HTTP Basic authentication for other APIs.
+
+=== User impersonation
+
```

Comment:
Great description again. I was wondering whether we should:

1. Describe the advantages of using user impersonation (audit logs in Redpanda will reveal the actual requester, unified authorization enforced by Redpanda)
2. On what APIs we can use impersonation (only kafka, schema registry and redpanda admin api)